extensions/cornerstone/src/tools/ImageOverlayViewerTool.tsx

```diff
@@ -0,0 +1,258 @@
+import { metaData, StackViewport, Types } from '@cornerstonejs/core';
+import { utilities } from '@cornerstonejs/core';
+import { BaseTool } from '@cornerstonejs/tools';
+import { guid } from '@ohif/core/src/utils';
+
+interface CachedStat {
+  color: number[];
+  overlays: (unknown & {
+    _id: string;
+    pixelDataRaw: ArrayBuffer;
+    type: 'G' | 'R';
+  })[];
+}
+
+/**
+ * Image Overlay Viewer tool is not a traditional tool that requires user interactin.
+ * But it is used to display Pixel Overlays. And it will provide toggling capability.
+ *
+ * The documentation for Overlay Plane Module of DICOM can be found in [C.9.2 of
+ * Part-3 of DICOM standard](https://dicom.nema.org/medical/dicom/2018b/output/chtml/part03/sect_C.9.2.html)
+ */
+class ImageOverlayViewerTool extends BaseTool {
+  static toolName = 'ImageOverlayViewer';
```

Comment:
Follow how we are naming tools in cs3d, this causes some weird error in build

`  static toolName;` 

`SegmentationDisplayTool.toolName = 'SegmentationDisplay';
export default SegmentationDisplayTool;
`

Reply:
Actually that is exactly how we are naming things - it is an equivalent line to the one you gave but is typescripty rather than javascripty.